cmd/kubectl-gadget/utils/flags.go

```diff
@@ -133,6 +137,12 @@ func AddCommonFlags(command *cobra.Command, params *CommonFlags) {
 			}
 		}
 
+		// Columns
+		if columns, err := command.PersistentFlags().GetString("columns"); err == nil && columns != "" {
+			// Fallback
+			command.PersistentFlags().Set("output", commonutils.OutputModeCustomColumns+"="+columns)
+		}
```

Comment:
Yeah, the integration demo is not more than that, a demo. We definitely have to talk about how to integrate it properly with the least code possible. I've also added a description field to the column info, so it would be great to also have an overview of the columns together with a short explanation on `--help`. That and all the new features like sorting, filtering etc (which also partly have to be added to the gadget side of things, then). I'll gladly do a more thorough integration example once the API of the package is stable enough.